code/headers/mq_sensors_gas_curve_c.hpp

```diff
@@ -18,8 +18,7 @@ namespace r2d2::gas_detection {
     public:
         /**
          * Multiple constructors to create an object with the gas id and gas
-         *curve
-         *
+         * curve
          **/
         mq_sensors_gas_curve_c(const int &gas_id,
```

Comment:
Please comment the parameters.

Reply:
@YouriMulder I think this is self explanetory, since this is the constructor and the class documentation has already explained what it does.